appveyor.yml

```diff
@@ -84,23 +85,23 @@ after_deploy:
             if ($ENV:APPVEYOR_REPO_TAG  -eq "true" -and $ENV:CONFIGURATION  -eq "Release" )
             {
                 $package_json = "C:\projects\inexor-game\package.json"
-                
-                
+
+
                 $package_version = $ENV:APPVEYOR_REPO_TAG_NAME -replace "-alpha"
-                
+
```

Comment:
this will break stuff
the bad lines are there for a reason.
and yes the reason is yaml sucks.